plugins/UsersManager/UsersManager.php

```diff
@@ -170,16 +171,21 @@ public static function checkPassword($password)
          *
          * @param string $password Checking password in plain text.
          */
+        $error = null;
+
         Piwik::postEvent('UsersManager.checkPassword', array($password));
```

Comment:
you need to add a try catch around that. The event is supposed to throw an exception if the password isn't valid, which would currently pass through even if `$returnError = true`

Reply:
@sgiehl that makes more sense, update it.